libs/npm/core/src/components/flex/col/styled/Col.tsx

```diff
@@ -2,7 +2,9 @@ import styled from 'styled-components';
 
 import { IColProps } from '..';
 
-export const Col = styled.div<IColProps>`
+export const Col = styled.div.attrs<IColProps>(({ className }) => ({
```

Comment:
@jdtoombs check out this syntax that allows us to correctly apply inline attributes to a styled component.  Before adding this any attributes we included in our code were being ignored.